tests/command_tests/load_test.rs

```diff
@@ -3,6 +3,44 @@ include!("../common/fs.rs");
 include!("../common/log.rs");
 
 pub fn test() {
+    check_github_load();
+    check_gitlab_load();
+}
+
+pub fn check_gitlab_load() {
+    utils::init_tpy();
+
+    utils::run_successfully("tpy load https://gitlab.com/api/v4/projects/cophilot%2Ftemplify-vault/repository/tree?path=Test");
```

Comment:
I see that only direct api calls are supported. Would it be possible to do it as well with the "normal url" of gitlab (e.g. "https://gitlab.com/cophilot/templify-vault/-/tree/main/Test")? Maybe parse the "normal url" to the api url? TBH not sure how easy that is...

Reply:
Thats a wonderful idea, @cophilot and I am more than happy to implement this no matter how easy or difficult it will be, as it will give more flexibility to the user while loading templates from gitlab. But I am thinking to take this as an enhancement in a separate PR. Will that be fine with you ?